src/muse.c

```diff
@@ -2055,6 +2060,12 @@ mloot_container(
                 break; /* out of takeout_count loop */
         } /* can_carry */
     } /* takeout_count */
+    /* Monster rummaged through a bag but found nothing of interest, possibly because
+       some items vanished. */
+    if (!res && vismon) {
+        pline("%s %s inside %s.", Monnam(mon), mon->mcansee ? "looks" : "feels about", 
```

Comment:
Formatting thing, this line and the above comment is longer than 80 characters

Reply:
Ugh, thanks. I'm not in my usual editor.